dbms/src/Server/Server.cpp

```diff
@@ -178,27 +178,33 @@ class TCPConnectionFactory : public Poco::Net::TCPServerConnectionFactory
 };
 
 
-int Server::main(const std::vector<std::string> & args)
+static std::string getCanonicalPath(std::string && path)
 {
-	Logger * log = &logger();
-
-	std::string path = config().getString("path");
 	Poco::trimInPlace(path);
 	if (path.empty())
 		throw Exception("path configuration parameter is empty");
 	if (path.back() != '/')
 		path += '/';
+	return path;
+}
+
+
+int Server::main(const std::vector<std::string> & args)
+{
+	Logger * log = &logger();
```

Comment:
It's a matter of taste-)
It was strange to me that main server function begins with explicit low-level string manipulations.
Reverted.